pentasharp/Controllers/DashboardController.cs

```diff
@@ -0,0 +1,28 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace WebApplication1.Controllers
+{
+    public class DashboardController : Controller
+    {
```

Comment:
Consider separating these controllers for better separation of concerns and maintainability. Meaning BusManagement and CreateBusCompany would have their own controller and so would TaxiManagement and TaxiManagement